app/code/Magento/Checkout/Block/Checkout/AttributeMerger.php

```diff
@@ -316,20 +316,19 @@ protected function getMultilineFieldConfig($attributeCode, array $attributeConfi
     protected function getDefaultValue($attributeCode)
     {
         switch ($attributeCode) {
```

Comment:
Hahaha, brilliant... I guess programming for 12 hours straight is not paying off ;)